File: 8801.c
#include <stdio.h>

int main(void) {
    int cnt = 0;

    for (int i = 1; i <= 3; i++){
        for (int j = 1; j <= 5; j++) {
            for (int k = 1; k <= 3; k++) {
                for (int l = 1; l <= 5; l++) {
                    if ((i == j) != (k == l)) {
                        cnt++;
                    }
                }
            }           
        }
    }

    printf(cnt);

    return 0;
}

File: 8801.py
cnt = 0

for i in range(1, 4):
    for j in range(1, 6):
        for k in range(1, 4):
            for l in range(1, 6):
                print('A: {}, {}, B: {}, {}'.format(i, j, k, l), end = '')
                
                if (i == j) != (k == l):
                    cnt += 1
                    print(', {}번째'.format(cnt))
                else:
                    print()